draft-ietf-mls-protocol.md

```diff
@@ -2769,14 +2771,27 @@ Some uses of resumption PSKs might call for the use of PSKs from historical
 epochs. The application SHOULD specify an upper limit on the number of past
 epochs for which the `resumption_psk` may be stored.
 
-## State Authentication Codes
-
-The main MLS key schedule provides a per-epoch `authentication_code`.
-If one of the parties is being actively impersonated by an attacker, their
-`authentication_code` will differ from that of the other group members.  Thus,
-members of a group MAY verify out of band that their `authentication_code`
-values are all the same, in order to ensure that they share the same view of the
-group.
+## Epoch Authenticators
+
+The main MLS key schedule provides a per-epoch `epoch_authenticator`. If one
+member of the group is being impersonated by an active attacker, the
+`epoch_authenticator` computed by their client will differ from those computed
+by the other group members.
+
+This property can be used to construct defenses against impersonation attacks
+that are effective even if members' signature keys are compromised. As a
+trivial example, if the users of the clients in an MLS group were to meet in
+person and confirm that their epoch authenticator values were equal, then each
+user would be assured that the others were not being impersonated in the current
+epoch. As soon as the epoch changed, though, they would need to re-do this
+confirmation. The state of the group would have changed, possibly
+introducing an attacker.
+
+More generally, in order for the members of an MLS group to obtain concrete
+authentication protections using the `epoch_authenticator`, they will need to
+use it in some secondary protocol (such as the face-to-face protocol above).
```

Comment:
Corresponding to the above edit:
"...secondary protocol (such as a face-to-face protocol)..."